docs/creating_content_basic.md

```diff
@@ -105,10 +105,9 @@ You can select a SiteAccess to preview by using a drop-down in the preview scree
 
 To keep your content organized and easy to find, you can add taxonomy entries to a Content item while creating or editing it.
 
-1. Go to **Taxonomy entries** section.
+1. Go to section with tag field.
```

Comment:
There is no specific section now, I think?

Reply:
No, as the "tag field" now can be named according to user needs.